docs/devs/e2e-tests.md

```diff
@@ -0,0 +1,58 @@
+# E2E Tests
+
+## Introduction
+    E2E testing is a critical aspect of Ozone HIS that ensures all the workflows work as expected.
+
+## Prerequisites
+    Before running Ozone E2E tests, ensure you have Node.js and npm installed on your machine.
+
+## Setup Instructions
+    Clone Ozone E2E repository: git clone https://github.com/ozone-his/ozone-e2e.
+    Navigate to the project directory: cd ozone-e2e.
+    Install dependencies: yarn install.
+
+## Test Case Documentation
+    Test Scenario: Patient with drug order becomes customer in Odoo.
+    Steps:
+        Login to O3, create a patient and start a visit.
+        Make a drug order.
+        Navigate to Odoo using Single Sign-On (SSO).
+    Expected Outcome: Patient is synced as a customer in Odoo.
+
+## Test Suite Structure
+    Test suite is organized into folders based on functionality.
+    ```
+    e2e
+    |__ tests
+    |   ^ Contains test cases.
+    |__ utils
+    |   ^ Contains utilities needed to set up and tear down tests,
+    |     as well as methods required by the tests to run.
+    ```
+
+## Configuration Options
+    Environment Settings: Use `.env` file to configure test environment (e.g., E2E_BASE_URL).
+    Browser Selection: Use the `--browser` flag to specify the browser (e.g., chrome, firefox).
+
+## Running Tests
+    Run all tests: npm playwright tests.
+    Run tests in a specific file: npm playwright test <file name>.
+    Run tests with custom configuration: npm playwright test --project=<browser name>.
```

Comment:
The commands for running tests are prefixed with `npm playwright`, which is not a standard npm command. If a custom npm script is being used, it should be defined in the `package.json` file and the command to run it would be `npm run <script-name>`. Or did you want to use `npx` instead of `npm`? `npx playwright tests`

Reply:
It is supposed to be `npx playwright test`